### DISCUSSION TITLE
Security onion stuck at Running post-installation scripts

### DISCUSSION THREAD

**Discussion by gangalal** [Security-Onion-Solutions/securityonion#12299] [2.4] [answered]
### Version

2.4.40

### Installation Method

Security Onion ISO image

### Description

installation

### Installation Type

Eval

### Location

on-prem with Internet access

### Hardware Specs

Exceeds minimum requirements

### CPU

10

### RAM

32

### Storage for /

400

### Storage for /nsm

400

### Network Traffic Collection

other (please provide detail below)

### Network Traffic Speeds

more than 10Gbps

### Status

No, one or more services are failed (please provide detail below)

### Salt Status

No, there are no failures

### Logs

No, there are no additional clues

### Detail

I am using dell laptop. It stuck at "Running post-installation scripts". Is there any solution?
![image](https://github.com/Security-Onion-Solutions/securityonion/assets/38142529/c8242957-e79e-47cd-bf57-5578a70a8ca3)

### Guidelines

- [X] I have read the discussion guidelines at https://github.com/Security-Onion-Solutions/securityonion/discussions/1720 and assert that I have followed the guidelines.

**Comment by cm-ops** [comment]
How was the ISO mounted? If you hit` alt+tab F2` and run `top`, what processes do you see running?

**Comment by TheAOWizard** [comment, marked as answer]
I had a server where it was also stuck at "Running post-installation scripts", I thought it would never progress and it looked like everything was stuck.
I let it be over night and 5hours later it was done with that step and functioned normal.

The server was over-spec'ed but had huge disks, so I have a theory that if you have some TB disks in size, this step just takes a looong time.
